package.json

```diff
@@ -105,33 +105,36 @@
     "webpack-bundle-analyzer": "^3.3.2",
     "webpack-cli": "^3.3.1",
     "webpack-dev-server": "^3.3.1",
+    "webpack-node-externals": "^1.7.2",
     "webpack-visualizer-plugin": "^0.1.11",
-    "yn": "^3.0.0"
+    "yn": "^3.0.0",
+    "wpt": "https://github.com/Woopra/js-client-tracker/archive/v5.1.19.tar.gz"
   },
   "dependencies": {
     "@babel/runtime-corejs3": "^7.4.3",
-    "blueimp-load-image": "2.12.2",
-    "classnames": "2.2.5",
+    "blueimp-load-image": "~2.12.2",
+    "classnames": "~2.2.5",
     "enumerate-devices": "^1.1.0",
-    "eventemitter2": "2.1.3",
-    "history": "4.5.1",
+    "eventemitter2": "~2.1.3",
+    "history": "~4.5.1",
     "node-polyglot": "^2.2.2",
-    "parse-unit": "1.0.1",
-    "pdfobject": "2.0.201604172",
+    "parse-unit": "~1.0.1",
+    "pdfobject": "~2.0.201604172",
     "preact": "^8.4.2",
     "preact-compat": "^3.18.4",
     "preact-context": "^1.1.3",
     "raven-js": "^3.26.3",
     "react-modal-onfido": "^1.6.1",
     "react-native-listener": "1.0.1",
     "react-phone-number-input": "^0.15.2",
-    "react-redux": "4.4.6",
+    "react-redux": "~4.4.6",
     "react-webcam-onfido": "^0.1.12",
-    "redux": "3.5.2",
-    "reselect": "2.5.1",
+    "redux": "~3.5.2",
+    "reselect": "~2.5.1",
     "socket.io-client": "^2.0.4",
-    "supports-webp": "1.0.3",
-    "visibilityjs": "1.2.4",
-    "wpt": "https://github.com/Woopra/js-client-tracker/archive/v5.1.19.tar.gz"
+    "supports-webp": "~1.0.3",
+    "visibilityjs": "~1.2.4",
+    "react": "^15.0.0",
```

Comment:
I don't think we call this directly, right? If so, I don't think it needs to go on the dependency lit

Reply:
for summary: this impacts the npm artifact in 0.2 MB. this is already a big artifact, so we will proceed until we consolidate react and preact